index.html

```diff
@@ -3,16 +3,16 @@
 <head>
 	<link rel="stylesheet" href="styles.css">
 	<meta charset="UTF-8">
-	<title>Microverse trials</title>
+	<title>Microverse trials review</title>
 </head>
 <body>
 <main id="main">
 	<div id="title-container">
```

Comment:
Excellent work follow the instructions but, kindly ask you to use semantic tags for every component of your code, which will help other developers to understand better your work.